codespell_lib/data/dictionary.txt

```diff
@@ -15749,7 +15749,7 @@ ist->is, it, its, it's, list,
 istalling->installing
 istance->instance
 istead->instead
-istener->listeners
+istener->listener
```

Comment:
Can we add the plural version too please:
`isteners->listeners`
Also probably the -ed and -ing versions (the latter certainly exists in grep.app).

Reply:
@peternewman I'd be happy to submit those changes but I think it should be a different PR (or at least a separate commit). Do you agree?

Reply:
Yeah another commit is fine (and my preference as GitHub plays better then).

Reply:
done in second commit